### PR TITLE
Feat/1545 (part 2) Setup deployment for live comms site

### LINKED ISSUE #1545: Migrate Design's Live Comms Site into the TANF-app repo.
**Description:**
Design is experimenting with a move to a web format rather than a PDF format for live comms to take advantage of accessibility improvements offered by web that don't come as simply in PDF form.

**Acceptance Criteria:** 
_Create a list of functional outcomes that must be achieved to complete this issue_

- [ ] The existing live comms repo is moved from Miles' repo to an existing project repo (TANF-app)
- [ ] Design is still able to maintain the live comms site as they do on its current GitHub Pages hosting (e.g. no/minimal new code introduced to make it work, control over dependencies)
- [ ] Hosted via cloudfoundry cloud.gov route
- [ ] Testing Checklist has been run and all tests pass
- [ ] README is updated, if necessary

**Tasks:**
_Create a list of granular, specific work items that must be completed to deliver the desired outcomes of this issue_ 

- [ ] Refactor html files for general purpose static use (currently tied to github website) 
- [ ] setup static file Cloud foundry build pack for cloud.gov
- [ ] Add CI workflow for deploying changes
- [ ] Add commands to locally test changes via simple HTTP server on localhost
- [ ] Add documentation for local development and deployment of the application

**Notes:**
_Add additional useful information, such as related issues and functionality that isn't covered by this specific issue, and other considerations that will be helpful for anyone reading this_ 

- _Note 1_
- _Note 2_
- _Note 3_

**Supporting Documentation:**
_Please include any relevant log snippets/files/screen shots_

Current code:
Link to site: https://reitermb.github.io/TANF-Data-Portal-Updates/
Link to repo: https://github.com/reitermb/TANF-Data-Portal-Updates/blob/main/index.html

**Open Questions:**

### PR DESCRIPTION
**Pull request by riatzukiza**
### Summary of Changes

**blocked until #1555 has been merged** 

In order to actually deploy the site, we needed to set up a cloud foundry deployment script, a manifest file, and make changes to the circle CI config. There are also some small adjustments that have to be made to the HTML files in order for them to find resources.

Pull request closes #1545 
_Acceptance criteria as stated in the issue_

### How to Test
_List the steps to test the PR_
These steps are generic, please adjust as necessary.
```
cd tdrs-frontend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d
cd tdrs-backend && docker-compose -f docker-compose.yml -f docker-compose.local.yml up -d 
```

1. Open http://localhost:3000/ and sign in.
1. Proceed with functional tests as described herein.
1. Test steps should be captured in the demo GIF(s) and/or screenshots below.

> *Demo GIF(s) and screenshots for testing procedure*

### Deliverable 1: Accepted Features

> *Performance Standard(s): At the beginning of each sprint, the Product Owner and development team will collaborate to define a set of user stories to be completed during the sprint. Acceptance criteria for each story will also be defined. The development team will deliver code and functionality to satisfy these user stories.*

> *Acceptable Quality Level: Delivered code meets the acceptance criteria for each user story. Incomplete stories will be assessed and considered for inclusion in the next sprint.*

+ [x] Look up the acceptance criteria in the related issue; paste ACs below in checklist format.
+ [x] Check against the criteria:
  + [x] The existing live comms repo is moved from Miles' repo to an existing project repo (TANF-app)
  + [x] Design is still able to maintain the live comms site as they do on its current GitHub Pages hosting (e.g. no/minimal new code introduced to make it work, control over dependencies)
 + [x] Hosted via cloudfoundry cloud.gov route
 + [x] Testing Checklist has been run and all tests pass
 + [ ]  README is updated, if necessary
As Product Owner, @lfrohlich will decide if ACs are met.

### Deliverable 2: Tested Code

> *Performance Standard(s): Code delivered under the order must have substantial test code coverage. Version-controlled HHS GitHub repository of code that comprises products that will remain in the government domain.*

> *Acceptable Quality Level: Minimum of 90% test coverage of all code. All areas of code are meaningfully tested.*

+ Are all areas of code introduced in this PR meaningfully tested?
  + [ ] If this PR introduces backend code changes, are they meaningfully tested?
  + [ ] If this PR introduces frontend code changes, are they meaningfully tested?
+ Are code coverage minimums met?
  + [ ] Frontend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)
  + [ ] Backend coverage: ___ (see https://github.com/raft-tech/TANF-app README coverage dashboard)

### Deliverable 3: Properly Styled Code

> *Performance Standard(s): GSA 18F Front- End Guide*

> *Acceptable Quality Level: 0 linting errors and 0 warnings*

+ [ ] Are backend code style checks passing on CircleCI?
+ [ ] Are frontend code style checks passing on CircleCI?
+ [ ] Does this PR change any linting or CI settings?

### Deliverable 4: Accessible

> *Performance Standard(s): Web Content Accessibility Guidelines 2.1 AA standards*

> *Acceptable Quality Level: 0 errors reported using an automated scanner and 0 errors reported in manual testing*
+ [x] Does this PR complete the epic? 
+ [ ] Are links included to any other gov-approved PRs associated with epic?
+ [x] Does PR include documentation for Raft's a11y review? 
+ [x] Did automated and manual testing with @iamjolly and @ttran-hub using Accessibility Insights reveal any errors introduced in this PR?
    + [See the full Accessibility Assessment plan here.](https://github.com/HHS/TANF-app/blob/main/docs/Technical-Documentation/how-government-will-test-a11y.md)

### Deliverable 5: Deployed

> *Performance Standard(s): Code must successfully build and deploy into the staging environment.*

> *Acceptable Quality Level: Successful deployment by assigning a 'Deploy with CircleCI - {env}' label*

+ [x] Was the code successfully deployed via automated CircleCI process to development on Cloud.gov?

### Deliverable 6: Documented

> *Performance Standard(s): Summary of user stories completed every two weeks. All dependencies are listed and the licenses are documented. Major functionality in the software/source code is documented, including system diagram. Individual methods are documented inline in a format that permits the use of tools such as JSDoc. All non-inherited 800-53 system security controls are documented in the Open Control or OSCAL format and HHS Section 508 Product Assessment Template (PAT) are updated as appropriate.*

> *Acceptable Quality Level: Combination of manual review and automated testing, if available*

+ [ ] If this PR introduces backend code, is that code documented both inline and overall?
+ [ ] If this PR introduces frontend code, is that code documented both inline and overall?
+ [ ] If this PR introduces dependencies, are their licenses documented?

### Deliverable 7: Secure

> *Performance Standard(s): Open Web Application Security Project (OWASP) Application Security Verification Standard 3.0*

> *Acceptable Quality Level: Code submitted must be free of medium- and high-level static and dynamic security vulnerabilities*

+ [ ] Does the OWASP Scan pass on CircleCI?
+ [ ] Do manual code review and manual testing detect any security issues?

### Deliverable 8: Context

> * Performance Standard(s): Code must be understandable and contextualized for the reviewers possess the knowledge and background necessary for analysis and constructive criticism to take place.

> * Acceptable Quality Level: Code submitted in the pull request has context.*
+ [ ] Does this pull request contain sufficient inline comments providing relevant context for code snippets?
+ [ ] Is the code understandable and lucid?
+ [ ] Does this pull request provide background for why coding decisions were made?
+ [ ] Can you as a reviewer explain and take ownership of these elements presented in this code review?

### CODE REVIEW

File: live-comms/August-2021-Update.html [hunk 1]
```diff
@@ -62,7 +62,7 @@
     </div>
     <nav aria-label="Primary navigation" class="usa-nav">
       <button class="usa-nav__close">
-        <img src="./close.svg" role="img" alt="Close" />
+        <img src="././img/close.svg" role="img" alt="Close" />
```
Comment (riatzukiza) [inline live-comms/August-2021-Update.html:65]: routing is handled differently by github than in a typical static site, some extra magic was happening

File: scripts/deploy-live-comms.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
+
+#!/bin/sh
+
+# source deploy-util.sh
```
Comment (abottoms-coder) [inline scripts/deploy-live-comms.sh:4]: Should this be removed? I don't see this file in your branch.

Comment (riatzukiza) [inline scripts/deploy-live-comms.sh:4]:
```suggestion
```

File: live-comms/index.html [hunk 3]
```diff
@@ -95,7 +95,7 @@
     </div>
     <nav aria-label="Primary navigation" class="usa-nav">
       <button class="usa-nav__close">
-        <img src="./close.svg" role="img" alt="Close" />
+        <img src="././img/close.svg" role="img" alt="Close" />
```
Comment (riatzukiza) [inline live-comms/index.html:98]:
```suggestion
        <img src="./img/close.svg" role="img" alt="Close" />
```

File: live-comms/August-2021-Update.html (hunk 1 above)
Comment (riatzukiza) [inline live-comms/August-2021-Update.html:65]:
```suggestion
        <img src="./img/close.svg" role="img" alt="Close" />
```

File: scripts/deploy-live-comms.sh (hunk 2 above)
Comment (riatzukiza) [reply scripts/deploy-live-comms.sh]: Removed it, I'll remove it from the other deploy scripts too. I copied it as a template. Maybe worth thinking about abstracting the deployment behavior after we've now done 3 different deployments, and we'll be talking about even more of that as we consider putting nginx infront of django? Part of the spike probably to think about it.

**Review by jorgegonzalez — APPROVED** [review]
LGTM

File: scripts/deploy-live-comms.sh [hunk 4]
```diff
@@ -0,0 +1,32 @@
+
+#!/bin/sh
+
```
Comment (riatzukiza) [inline scripts/deploy-live-comms.sh:3]: Add documentation for this script either in scripts/readme or move it to live comms (?)

**Comment by riatzukiza** [comment]
Add to main readme documentation about the live-comms static app

**Comment by riatzukiza** [comment]
Remove top nav

File: live-comms/August-2021-Update.html [hunk 5]
```diff
@@ -54,7 +54,7 @@
     <div class="usa-navbar">
       <div class="usa-logo" id="basic-logo">
         <em class="usa-logo__text">
-          <a href="#main-content" title="TDP Project Updates">
+          <a href="/index.html" title="TDP Project Updates">
```
Comment (ADPennington) [inline live-comms/August-2021-Update.html:57]: @riatzukiza can you confirm that this line of code reflects the change you mentioned in tabletop?

File: live-comms/README.md [hunk 6]
```diff
@@ -1 +1,23 @@
 # Design Environment for web hosted live comms
+
+The live comms website is for communicating
```
[the remark this answers is not in the document]
Comment (reitermb) [reply live-comms/README.md]: @ADPennington Just committed a small update including a few changes there Aaron and I co-worked that got lost somewhere along the line.

File: live-comms/August-2021-Update.html [hunk 7]
```diff
@@ -12,7 +12,7 @@
   </head>
   <body>
 
-    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
+    <a class="usa-skipnav" href="./index.html">Skip to main content</a>
```
Comment (riatzukiza) [inline live-comms/August-2021-Update.html:15]:
```suggestion
    <a class="usa-skipnav" href="/index.html">Skip to main content</a>
```

File: live-comms/August-2021-Update.html [hunk 8]
```diff
@@ -63,21 +63,21 @@
     </div>
     <nav aria-label="Primary navigation" class="usa-nav">
       <button class="usa-nav__close">
-        <img src="./close.svg" role="img" alt="Close" />
+        <img src="./img/close.svg" role="img" alt="Close" />
       </button>
       <ul class="usa-nav__primary usa-accordion">
         <li class="usa-nav__primary-item">
-          <a href="#main-content" class="usa-nav__link">
+          <a href="./index.html" class="usa-nav__link">
```
Comment (riatzukiza) [inline live-comms/August-2021-Update.html:70]:
```suggestion
          <a href="/index.html" class="usa-nav__link">
```

File: live-comms/June-2021-Update.html [hunk 9]
```diff
@@ -63,21 +63,21 @@
     </div>
     <nav aria-label="Primary navigation" class="usa-nav">
       <button class="usa-nav__close">
-        <img src="./close.svg" role="img" alt="Close" />
+        <img src="././img/close.svg" role="img" alt="Close" />
```
Comment (riatzukiza) [inline live-comms/June-2021-Update.html:66]:
```suggestion
        <img src="./img/close.svg" role="img" alt="Close" />
```

File: live-comms/June-2021-Update.html [hunk 10]
```diff
@@ -63,21 +63,21 @@
...
       </button>
       <ul class="usa-nav__primary usa-accordion">
         <li class="usa-nav__primary-item">
-          <a href="#main-content" class="usa-nav__link">
+          <a href="./index.html" class="usa-nav__link">
```
Comment (riatzukiza) [inline live-comms/June-2021-Update.html:70]:
```suggestion
          <a href="/index.html" class="usa-nav__link">
```

File: live-comms/June-2021-Update.html [hunk 11]
```diff
@@ -95,13 +95,13 @@
         <nav aria-label="Secondary navigation">
           <ul class="usa-sidenav">
             <li class="usa-sidenav__item">
-              <a href="#main-content">January 2022</a>
+              <a href="./index.html">January 2022</a>
```
Comment (riatzukiza) [inline live-comms/June-2021-Update.html:98]:
```suggestion
              <a href="/index.html">January 2022</a>
```

File: live-comms/June-2021-Update.html [hunk 12]
```diff
@@ -63,21 +63,21 @@
...
            <li class="usa-nav__primary-item">
-          <a href="August-2021-Update" class="usa-nav__link">
+          <a href="August-2021-Update.html" class="usa-nav__link">
             <span>August 2021</span>
           </a>
         </li>
            <li class="usa-nav__primary-item">
-          <a href="June-2021-Update" class="usa-current" aria-current="page">
+          <a href="June-2021-Update.html" class="usa-current" aria-current="page">
             <span>August 2021</span>
```
Comment (ADPennington) [inline live-comms/June-2021-Update.html:81]:
```suggestion
            <span>June 2021</span>
```

**Review by ADPennington — CHANGES_REQUESTED** [review]
@riatzukiza @reitermb one suggestion below before this is ready to go. looks great!

**Review by ADPennington — APPROVED** [review]
thanks @riatzukiza @reitermb @ttran-hub 👍🏾

**Pull request merged**